src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/content-browser.component.ts

```diff
@@ -31,19 +32,21 @@ import { ContentDownloadDialogService } from '../dialog/components/content-downl
   ]
 })
 export class ContentBrowserComponent extends BaseBrowserComponent<Content> implements OnChanges {
-
   @Input()
   public channel: Channel | null = null;
 
+  public get deviceBreakpoint(): boolean {
```

Comment:
Add to BaseBrowserComponent as this needs to be reused for other browsers.

Reply:
@neotek done